docs/usage/secrets_rotation.md

```diff
@@ -0,0 +1,19 @@
+# Secrets and rotation
+
+## List of secrets
+
+**Cloudprovider Secret**
+
+*Example*: https://github.com/gardener/gardener/blob/master/example/70-secret-provider.yaml
+
+*Usage*: Authenticate gardener and kubernetes components for infrastructure operations
+
+*Rotation*: Rotating the cloudprovider secret requires multiple steps:
+
+1. Either create the new secret in the garden cluster and add a secretBinding or edit the current secret.
+2. :warning: Wait until all Shoots using the secret are reconciled before you deactivate the old secret! Otherwise the shoots will no longer function.
+3. After you are sure all Shoots using the secret are reconciled you can go ahead and deactivate the old secret in your infrastructure provider.
```

Comment:
Can you make it more clear what you mean with "deactivate the old secret"?

Reply:
adapted it PTAL